## Overview
Affected versions of [`zope2`](https://pypi.python.org/pypi/Zope2) are vulnerable to Denial of Service (DoS, crash of worker threads) via vectors that trigger uncaught exceptions.

## References
- [NVD](https://nvd.nist.gov/vuln/detail/CVE-2010-3198)
- [Zope Version Announcement](https://mail.zope.org/pipermail/zope-announce/2010-September/002247.html)
